lukimgather/tests/__init__.py

```diff
@@ -1,14 +1,62 @@
+import secrets
+import string
+
+from django.conf import settings
+from django.contrib.auth import authenticate
 from django.test import RequestFactory
 from django.urls import reverse
 from graphene_django.utils.testing import GraphQLTestCase
+from graphql_jwt.shortcuts import get_token
 from model_bakery import baker, random_gen
 
 
 class TestBase(GraphQLTestCase):
     baker = baker
     baker.generators.add("lukimgather.fields.LowerCharField", random_gen.gen_string)
     baker.generators.add("lukimgather.fields.LowerEmailField", random_gen.gen_email)
-    baker.generators.add("ckeditor.fields.RichTextField", random_gen.gen_text)
+    baker.generators.add("ckeditor.fields.RichTextField", random_gen.gen_string)
     factory = RequestFactory()
     fixtures = ["support/content/email.yaml"]
     GRAPHQL_URL = reverse("api")
+
+    @staticmethod
+    def make_random_password(length=20):
+        alphabet = string.ascii_uppercase + string.ascii_lowercase + string.digits
+        password = "".join(secrets.choice(alphabet) for i in range(length))
+        return password
+
+    @classmethod
+    def setUpClassInit(
+        self,
+        is_active=True,
+        is_staff=False,
+        is_superuser=False,
+        _quantity=1,
+    ):
+        if not all(isinstance(i, bool) for i in [is_active, is_staff, is_superuser]):
+            raise TypeError("User status flags must be boolean")
+        if not isinstance(_quantity, int) or _quantity < 1:
+            raise ValueError("Quantity must be a positive integer")
```

Comment:
This may not be required if baker.make raise error. I was only asking if baker.make do not reject negative quantity below code will raise error. Only add this logic in those cases